package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@algoan/react-eslint-config",
```

You should respect [ESLint shareable config convention](https://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config)

```suggestion
  "name": "@algoan/eslint-config-react",
```

---

Thanks @ccoeurderoy , I didn't see this part of the documentation.

But I would say that if I change the package's name, I should rename this repository too, to be consistent (or delete and create new one?). What do you think?

---

Yes you can simply rename it 😄

---

Ok, even if this repository is public, I suppose nobody has cloned it 😄

---

Even if it was cloned, github automatically adds a redirection when you rename a repository.

---

Even if it was the case, a redirection is made by GitHub 😉 so the user won't be impacted

---

( I was faster 😜 )

---

Smart!